### PR TITLE
Update Microsite CSV

### PR DESCRIPTION
**Pull request by jamesmcclain**
The microsite CSV has been regenerated with precipitation levels which are whole numbers of centimeters.

Connects #44

### CODE REVIEW

File: tr55/makeMiniAppTable.py [hunk 1]
```diff
@@ -20,7 +29,7 @@
     writer.writerow(('P', 'land', 'soil', 'ET', 'I', 'R'))
 
     # values of inputs to the model
-    precips = [0.5, 1.0, 2.0, 3.2, 8.0]
+    precip_cm = [1, 2, 3, 5, 8, 13, 21]
```
Comment (rajadain) [inline tr55/makeMiniAppTable.py]: :wink:

Comment (rajadain) [inline tr55/makeMiniAppTable.py]: Although, on second thought, we should probably only generate the ones we use, lest this misalignment cause consternation six months from now during a period of stress.

**Comment by rajadain** [comment]
+1, great job!

**Comment by jamesmcclain** [comment]
Thanks for reviewing!

**Pull request merged**